data-prepper-plugins/aggregate-processor/README.md

```diff
@@ -108,6 +108,33 @@ While not necessary, a great way to set up the Aggregate Processor [identificati
         { "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "aggr._count": 1, "aggr._start_time": "2022-11-05T23:28:31.916Z"}
       ```
 
+### <a name="histogram"></a>
```

Comment:
Can we add this link to the list of aggregate actions further up in this README?